src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.model;
+
+import core.basesyntax.enums.Color;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double sideA;
+    private final double sideB;
```

Comment:
Lets rename those fields to male them more informative 
for example (smallBase, bigBase), (top, Bottom) or something like that